src/libmongoc/src/mongoc/mongoc-error-private.h

```diff
@@ -85,4 +85,7 @@ _mongoc_error_is_state_change (bson_error_t *error);
 bool
 _mongoc_error_is_network (const bson_error_t *error);
 
+bool
+_mongoc_error_is_server (bson_error_t *error);
```

Comment:
Since `error` is not being modified, and is therefore not an out parameter, should this instead be `const bson_error_t *error`?